base/uk.ac.stfc.isis.ibex.opis/check_opi_format_tests.py

```diff
@@ -0,0 +1,558 @@
+import unittest
+from lxml import etree
+from check_opi_format import CheckOpiFormat
+
+
+class TestCheckOpiFormatMethods(unittest.TestCase):
+
+    def test_that_if_a_valid_led_tag_is_parsed_it_causes_no_errors(self):
+        # Arrange
+        checker = CheckOpiFormat()
```

Comment:
As this line is in every single test, you can instead move it into a setUp-method that automatically gets called before each test.